docs/login_gov.md

```diff
@@ -0,0 +1,70 @@
+# Challenge.Gov Login.Gov Setup
+
+Currently login is managed and configured for the portal and the platform. In the future, this platform will be the connection point for login, so this guide serves for that future.
+
+## Login.gov Configuration
+
+1. Go to the Login.gov dashboard:
+
+```
+# For Sandbox acccess in dev/test/staging
+https://dashboard.int.identitysandbox.gov/
+```
+
+1. Login, Access is granted by the login[dot]gov team.
+
+1. Under Apps section (nav top right), click “Create a new test app”
+
+1. Register the app environment
+
+```
+Friendly name - "Challenge <environment>"
+Description - <blank>
+Agency - GSA
+Team - Challenge.gov
+Identity protocol - openid_connect
+Identity verification level (IAL) - IAL1
+Issuer - `urn:gov:gsa:openidconnect.profiles:sp:sso:gsa:challenge_gov_<environment>`
+Logo - Upload the Challenge.gov Logo
+Public certificate - see additional steps below
+Return to App URL - <blank>
+Failure to Proof URL - <blank>
+Push Notification URL - <blank>
+Redirect URIs - use the following replacing the app name
+`https://your-app-name.app.cloud.gov/auth/result`
+Attribute bundle - email, first_name, last_name
+```
+
+## Certificate Management
+
+```
+# create new keys by running the line below in terminal.
+$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365
+```
+
+Paste the cert.pem into the field including the
+`-----BEGIN CERTIFICATE-----`
+`-----END CERTIFICATE-----`
+
+Then re-encrypt the key for the elixir app, run the following with the correct key names:
+`openssl rsa -aes128 -in key.pem -out 128_key.pem`
+
+### For Local Development
+
+The two files (cert and private key) must exist in the root of the app codebase.
+In this case the private key should not have a password.
+
+```
+private_key_path: "local_key.pem"
+public_key_path: "local_cert.pem"
+```
+
+### For Deployment Via CircleCI
```

Comment:
This section is a little confusing. I'm not sure what it means that CircleCI creates the files "just in time". How does it create a key that matches the one that's uploaded to login.gov's configuration? Also the `escape-eol.py` doesn't exist in the Rails repo.